paper/paper.md

```diff
@@ -0,0 +1,88 @@
+
+# Statement of need
+
+While other open-source agent-based models have been developed for studying energy markets, such as PowerACE [@bublitzAgentbasedSimulationGerman2014] and AMIRIS [@schimeczekAMIRISAgentbasedMarket2023], the possible integration of reinforcement learning methods into the behavioral strategies of market agents is currently unique to `ASSUME`.
+Most notably it enables the highest number of simultanelously learning market agents in literature [@miskiwExplainableDeepReinforcement2024].  
+This feature allows for the exploration of new market designs and emergent dynamics in energy markets using a common open-source simulation framework.
+Further unique features of `ASSUME` are the extensive market abstraction which allows to define complex multi-market scenarios as shown in [@maurerMarketAbstractionEnergy2023].
+`ASSUME` is designed to provide results which are easily comparable and interactable.
+This is set in contrast to other agent-based simulation frameworks, which are often designed to be used for a specific use case.
+Instead, various moving parts can be configured, such as the market abstraction, the agent properties and bidding strategies, or the reinforcement learning methods.
+
+# Architecture
+
+`ASSUME` builds on the open-source agent-based simulation library [mango-agents](https://pypi.org/project/mango-agents/) to model the simulation world and to give users continuous feedback during the simulation runs.
+Interaction with the results of the simulation is possible through the Grafana dashboards, which do have direct access to the simulation database.
+Additionally, writing the output to CSV files is supported for scenarios in which a database is not available or needed.
+
+New scenarios can be created by providing CSV files, accessing simulation data from a database or by using the object-oriented API, which makes it possible to integrate new classes, maket mechanisms and bidding strategies as well.
+
+![Basic Architecture of the ASSUME simulation\label{fig:architecture}](../docs/source/img/architecture.svg)
+
+For most of such adjustments, only small changes to the code are necessary.
+This is possible through the decoupled architecture which separates the declaration of simulation behavior from the input data, while allowing to define either externally without changes to the core.
+The overview of available classes and the interaction with the world is shown in \autoref{fig:architecture}.
+Extensive documentation about these features is available at [https://assume.readthedocs.io/en/latest/](https://assume.readthedocs.io/en/latest/).
```

Comment:
here maybe also add the availability of the tutorials to get started along the documentation